dali/operators/reader/loader/webdataset/tar_utils.h

```diff
@@ -61,6 +62,21 @@ class DLL_PUBLIC TarArchive {
    * @brief Returns whether it has reached the end of archive.
    */
   bool EndOfArchive() const;
+  /**
+   * @brief Sets the offset to which the stream pointer should go.
+   * @remark The offset must point to a file header; other values will cause undefined behaviour.
+   */
+  void Seek(size_t offset);
```

Comment:
Please think through the class documentation. Again, the `Seek` function is not listed there. Maybe it'd be better not to list the functions in the class doc, but put the necessary remarks to the function docs?

Reply:
done